common/app/model/Asset.scala

```diff
@@ -107,8 +107,9 @@ case class VideoAsset(
   // The video duration in seconds
   val duration: Int = fields.get("durationSeconds").getOrElse("0").toInt +
                            (fields.get("durationMinutes").getOrElse("0").toInt * 60)
+  val durationSeconds: Int = fields.get("durationSeconds").getOrElse("0").toInt
```

Comment:
Ha no worries. That's a good point on using the new vals for legibility.

I believe videojs needs the duration in seconds to do its magic